Recursive/test/ackerman.test.js

```diff
@@ -0,0 +1,23 @@
+import { ackerman } from '../Ackerman'
+
+describe('Ackerman', () => {
+  it('should return 1', () => {
```

Comment:
Please use `each` for your tests: https://github.com/TheAlgorithms/TypeScript/blob/master/CONTRIBUTING.md#writing-good-tests